packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -414,6 +458,28 @@ mixin SchedulerBinding on BindingBase {
     return entry.completer.future;
   }
 
+  /// Schedules the given `task` with the given `priority` and returns a
+  /// [Future] that completes to the `task`'s eventual return value.
+  ///
+  /// Like [scheduleTask] except that it handles async tasks.
+  ///
+  /// See also:
+  ///   * [scheduleTask] for a description of the priority system.
```

Comment:
nit: dartdoc template can be used to inline the description of the priority system.